### ISSUE TITLE
Bug 54108751: [React Native Gallery -> Checkbox]: Screen reader fails to announce the success info upon activating ‘Copy to clipboard’ button.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#495]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Checkbox’ button present under “Basic Input” group and invoke it.
Navigate to ‘Copy to clipboard’ button and invoke it.
Listen to narrator announcement. 
Actual Result:
Screen reader fails to announce the success info upon activating ‘Copy to clipboard’ button. When user invoke it narrator stays on mute.
When user invoke ‘Copy to clipboard' button, there is no visual change for the button.
 
Note: Same issue is observed in the entire application.
 
Expected Result:
Screen reader should announce the success info upon activating ‘Copy to clipboard’ button. When user invoke it narrator should announce as “Copied to clipboard”.
 
User Impact:
Users with visual impairments who rely on screen reader will not be able to understand if the narrator is not announcing the success message after activating the button and user will not get to know whether the performed action is completed or not.
 
Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54108751
